### PR TITLE
Added UI yaml files

### PR DESCRIPTION
**Pull request by raresgaia123**
## Description

Added UI yaml files.

## Type of Change

- [ ] Bug Fix
- [x] New Feature
- [ ] Breaking Change
- [ ] Refactor
- [ ] Documentation
- [ ] Other (please describe)

## Checklist

- [ ] I have read the [contributing guidelines](/CONTRIBUTING.md)
- [ ] Existing issues have been referenced (where applicable)
- [ ] I have verified this change is not present in other open pull requests
- [ ] Functionality is documented
- [ ] All code style checks pass
- [ ] New code contribution is covered by automated tests
- [ ] All new and existing tests pass

### CODE REVIEW

**Review by myungjin — COMMENTED** [review]
Please consider to address my comments.

File: fiab/helm-chart/control/templates/dashboard-ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            service:
+              name: {{ .Release.Name }}-dashboard
+              port:
+                number: {{ .Values.servicePort.dashboard }}
+          pathType: Prefix
+          path: /
+        {{ else }}
+        - backend:
+            serviceName: {{ .Release.Name }}-dashboard
+            servicePort: {{ .Values.servicePort.dashboard }}
```
[the remark this answers is not in the document]
Comment (raresgaia123) [reply fiab/helm-chart/control/templates/dashboard-ingress.yaml:45]: Fixed.

File: fiab/helm-chart/control/templates/dashboard-service.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    app.kubernetes.io/name: {{ .Release.Name }}-dashboard
+    tier: backend
+  name: {{ .Release.Name }}-dashboard
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+    - name: dashboard
+      protocol: TCP
+      port: {{ .Values.servicePort.dashboard }}
+      targetPort: {{ .Values.servicePort.dashboard }}
```
[the remark this answers is not in the document]
Comment (raresgaia123) [reply fiab/helm-chart/control/templates/dashboard-service.yaml:31]: Fixed.

**Review by myungjin — APPROVED** [review]
lgtm

**Pull request merged**